pkg/client/client.go

```diff
@@ -214,15 +214,12 @@ func (o *OptimizelyClient) GetFeatureVariableString(featureKey, variableKey stri
 // GetFeatureVariable returns feature as a string along with it's associated type
 func (o *OptimizelyClient) GetFeatureVariable(featureKey, variableKey string, userContext entities.UserContext) (value string, valueType entities.VariableType, err error) {
 
-	decisionContext, featureDecision, err := o.getFeatureDecision(featureKey, userContext)
+	_, featureDecision, err := o.getFeatureDecision(featureKey, variableKey, userContext)
```

Comment:
You should get the var from the context so you don't have to put it in the decision

Reply:
addressed